src/main/java/com/elixr/poc/rest/response/UserGetResponse.java

```diff
@@ -0,0 +1,16 @@
+package com.elixr.poc.rest.response;
+
+import com.elixr.poc.data.User;
+import jakarta.validation.constraints.Negative;
+import lombok.Builder;
+import lombok.Data;
+
+import java.util.List;
+
+@Data
+@Builder
+public class UserGetResponse {
```

Comment:
UserGetResponse & UserPostResponse has a common field boolean success
So its better to define an abstract class UserReponse with 1 attribute "protected boolean success"
Then UserGetResponse & UserPostResponse extends this new class

Reply:
Created a class UserResponse and extending it where needed